app/helpers/routes_helper.rb

```diff
@@ -1,4 +1,6 @@
 module RoutesHelper
+  include Placements::Routes::OrganisationsHelper
```

Comment:
My Rails helper & Zeitwerk knowledge is not sharp enough to know if/why this `include` is needed.

Other helper modules in the codebase aren't explicitly included – so I assume Rails autoloads them in. Why is this one different?

Reply:
It's nothing fancy, you're right that they're all available in the views by default but we're using them here:

https://github.com/DFE-Digital/itt-mentor-services/blob/03ca825ebbd1425170fa1f78d1edcb764a2d6f19/app/controllers/placements/organisations_controller.rb#L7-L13

In the original implementation we added the include here:

https://github.com/DFE-Digital/itt-mentor-services/blob/03ca825ebbd1425170fa1f78d1edcb764a2d6f19/app/controllers/application_controller.rb#L3

I wanted to keep the disruption to the code as minimal as possible as I prepared to refactor so I decided to add the include inside the existing `RoutesHelper` instead of adding a new one to `ApplicationController`.

Reply:
Makes sense! Thanks for explaining.